src/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.28)
+project(tl2_raytracer CXX)
+
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+add_library(Image)
+
+target_sources(Image PUBLIC
+	FILE_SET CXX_MODULES
+	BASE_DIRS ${CMAKE_SOURCE_DIR}/modules
+	FILES modules/image.cxx
+)
```

Comment:
This is perhaps fine for now, but be aware that libraries should have multiple directories. Having one library per file is a bit pointless; you want a library to group multiple files.